README.md

```diff
@@ -59,22 +59,42 @@ Ensure that Python directory is included in your `PATH` system environment varia
 4. Select the serial port associated with the board that you flashed the firmware on.
 5. Click 'Start'.
 
-## Wireshark configuration
+## Wireshark configuration for Thread
 
-### Capturing Thread packets
+### Decryption key
 
-In order to capture packets exchanged on the Thread network it is necessary to configure the Wireshark to use correct decryption keys.
+In order to decode packets exchanged on the Thread network it is necessary to configure the Wireshark to use correct decryption keys.
 To set decryption keys go to `Edit -> Preferences... -> Protocols -> IEEE 802.15.4 -> Decryption Keys`. The default decryption key used by nRF5 SDK for Thread and Zigbee examples is `00112233445566778899aabbccddeeff`. Set decryption key index to `0` and `Key hash` to `Thread hash` value.
 
-### Custom Wireshark dissector
+### CoAP port configuration
+
+Thread uses CoAP protocol on port 61631 for internal purposes. To correctly decode packets sent over that port you can either:
+* apply the setting globally by editing CoAP protocol settings in `Preferences` and changing the `CoAP UDP port` value to `61631`.
+* apply it on per-capture basis by adding entry to `Analyze -> Decode as...`. Set `Field`
+column to `UDP port`, `Value` to `61631` and decode using CoAP dissector (`Current` column).
+
+### 6loWPAN contexts
+
+Go to 6loWPAN settings in `Preferences` window and add values:
+* Context 0: `fdde:ad00:beef:0::/64`
+* Context 1: `fd11:22::/64`
+
```

Comment:
Add note that those contextes may vary depending of the Thread Network Data. Those values are default for our SDK.